From 4486a693942a80a0b37e5a1fb3b951a0fa39b951 Mon Sep 17 00:00:00 2001
From: Brian Dooley <brian.dooley@intel.com>
Date: Wed, 19 Jan 2022 17:55:44 +0000
Subject: [PATCH 3/7] Remove extra argument from pktmbuf_dump()

---
 src/platform/linux/cndp/udp.rs | 1 -
 1 file changed, 1 deletion(-)

diff --git a/src/platform/linux/cndp/udp.rs b/src/platform/linux/cndp/udp.rs
index d373060..2ea5743 100644
--- a/src/platform/linux/cndp/udp.rs
+++ b/src/platform/linux/cndp/udp.rs
@@ -632,7 +632,6 @@ impl CndpUdpCommon {
             let msg = "Dump pktmbuf\n";
             // Passing null ptr will dump to stdout.
             pktmbuf_dump(
-                std::ptr::null_mut() as *mut FILE,
                 get_cstring_from_str(msg).as_ptr(),
                 pkt_mbuf,
                 1500,
-- 
2.30.2

